.packit.yaml

```diff
@@ -33,6 +33,32 @@ packages:
         - bash -c "ls -1t ./fedora/*.tar.gz | head -n 1"
       get-current-version: python3 -m setuptools_scm
 
+  specfile-centos-stream:
```

Comment:
I'm a bit uncertain about the `centos-stream` name, it's rather ambiguous. When I hear CentOS Stream I think https://gitlab.com/redhat/centos-stream/rpms, and we also refer to it as such in our [docs](https://packit.dev/docs/configuration/upstream/propose_downstream#syncing-the-release-to-centos-stream). What about something like `specfile-centos-integration-sig`?